distribution/lib/Standard/Base/0.0.0-dev/src/Data/Ordering.enso

```diff
@@ -23,42 +23,38 @@ from project.Data.Boolean import all
 
     Should there be a need to redefine the default implementation, here is a way:
     Define conversion function `Comparable.from` for your `type` and return pointer to
-    another `type` that satisfies either of the following two definitions:
+    another `type` that satisfies the following definition:
 
     ```
-    type Ordered_Comparator T
-        is_ordered = True
+    type Comparator T
         compare : T -> T -> (Ordering|Nothing)
         hash : T -> Integer
-
-    type Unordered_Comparator T
-        is_ordered = False
-        equals : T -> T -> (Boolean|Nothing)
-        hash : T -> Integer
     ```
 
-    Or `Incomparable` in case that the type `T` should not be compared at all.
-
     Every type must provide exactly one comparator, i.e., any method of form
     `Comparable.from (_:My_Type)` must always return the same comparator type.
```

Comment:
Can we have two `from` conversions for the same type?

I think this should be more like
```suggestion
    Every type must provide exactly one comparator, i.e., the method of form
    `Comparable.from (_:My_Type)` must return the same comparator type for every value.
```

Reply:
Agreed. No, we cannot have more conversions for the same type.

Reply:
So please rephrase this.